component/go.mod

```diff
@@ -47,3 +67,25 @@ retract (
 	v0.76.0 // Depends on retracted pdata v1.0.0-rc10 module, use v0.76.1
 	v0.69.0 // Release failed, use v0.69.1
 )
+
+replace go.opentelemetry.io/collector/semconv => ../semconv
+
+replace go.opentelemetry.io/collector/processor => ../processor
+
+replace go.opentelemetry.io/collector/config/confignet => ../config/confignet
+
+replace go.opentelemetry.io/collector/connector => ../connector
+
+replace go.opentelemetry.io/collector/service => ../service
+
+replace go.opentelemetry.io/collector/exporter => ../exporter
+
+replace go.opentelemetry.io/collector => ../
+
+replace go.opentelemetry.io/collector/receiver => ../receiver
+
+replace go.opentelemetry.io/collector/extension/zpagesextension => ../extension/zpagesextension
+
+replace go.opentelemetry.io/collector/extension => ../extension
+
+replace go.opentelemetry.io/collector/consumer => ../consumer
```

Comment:
We don't need the replaces anymore, right?

Reply:
this is the result from running `make crosslink` 🤷🏻

Reply:
re-ran crosslink after rebasing from main, these added lines are gone